doc/src/manual/style-guide.md

```diff
@@ -447,3 +447,43 @@ julia> h(1)
 
 Thus, use `Int` literals when possible, with `Rational{Int}` for literal non-integer numbers,
 in order to make it easier to use your code.
+
+## Workarounds for named positional arguments
+Named positional arguments are not supported in Julia, however, some users may still wish to use a named positional arguments code-style.
+There are several workarounds that can provide an experience similar to named positionals.
+Consider the `mod` method, which a user might call with the ambiguous variables `a` and `b` in the positional arguments:
```

Comment:
again, don't call it "ambiguous." focus on the _positive_ pattern of documenting code rather than the implied _negative_ pattern of using (anonymous) positional arguments